eth/catalyst/api_test.go

```diff
@@ -84,7 +84,7 @@ func generateMergeChain(n int, merged bool) (*core.Genesis, []*types.Block) {
 	generate := func(i int, g *core.BlockGen) {
 		g.OffsetTime(5)
 		g.SetExtra([]byte("test"))
-		tx, _ := types.SignTx(types.NewTransaction(testNonce, common.HexToAddress("0x9a9070028361F7AAbeB3f2F2Dc07F82C4a98A02a"), big.NewInt(1), params.TxGas, big.NewInt(params.InitialBaseFee*32), nil), types.LatestSigner(&config), testKey)
+		tx, _ := types.SignTx(types.NewTransaction(testNonce, common.HexToAddress("0x9a9070028361F7AAbeB3f2F2Dc07F82C4a98A02a"), big.NewInt(1), params.TxGas, big.NewInt(params.InitialBaseFee*26), nil), types.LatestSigner(&config), testKey)
```

Comment:
Why does this have to be changed to 26?

Reply:
@ssandeep, I have a comment in the code for the test, reporting it here 
```
	Given the introduction of `BorDefaultTxPoolPriceLimit=30gwei`,
	we set `testTxPoolConfig.PriceLimit = 1` to avoid rewriting all `legacypool_test.go` tests,
	causing code divergence from geth, as this has been widely tested on different networks.
	Also, `worker_test.go` has been adapted to reflect such changes.
	Furthermore, config test can be found in `TestTxPoolDefaultPriceLimit`
```
Basically we adopted some workarounds not to change the whole structure of tests.
In this one specifically, with `fee=1`, `geth` used `2` to make the test pass.
With `30gwei` I just modified it to be `32`.
So now with `25`, changing it to something just higher than it would work.